models/service_model.go

```diff
@@ -99,6 +99,15 @@ func (s *Service) SetCustomLabels(m map[string]string) error {
 	return setLabels(m, &s.CustomLabels)
 }
 
+// GetDatabaseNameOrDefault retrieves database name.
+func (s *Service) GetDatabaseNameOrDefault() string {
```

Comment:
I think it's better to add new migration instead of this hack

Reply:
Agree, it is ugly. I will rewrite tests.